resources/js/views/settings/Application.vue

```diff
@@ -256,6 +256,187 @@
           </template>
         </b-form-group>
 
+        <hr>
+
+        <!-- Banner -->
+        <b-form-group
+          label-class="font-weight-bold"
+          :state='fieldState("banner")'
+          :label="$t('settings.application.banner.title')"
+          ref='banner-form-group'
+        >
+          <b-form-group
+            :state='fieldState("banner.enabled")'
+          >
+            <b-form-checkbox
+              id='banner-enabled'
+              v-model='settings.banner.enabled'
+              :state="fieldState('banner.enabled')"
+              :disabled='isBusy || viewOnly || !loaded'
+              switch
+            >
+              {{ $t('settings.application.banner.enabled') }}
+            </b-form-checkbox>
+
+            <template slot='invalid-feedback'>
+              <div v-html="fieldError('banner.enabled')"></div>
+            </template>
+          </b-form-group>
+
+          <banner
+            class='mt-4'
+            v-if='settings.banner.enabled'
+            :background='settings.banner.background'
+            :color='settings.banner.color'
+            :enabled='settings.banner.enabled'
+            :icon='settings.banner.icon'
+            :link='settings.banner.link'
+            :message='settings.banner.message'
+            :title='settings.banner.title'
+          ></banner>
+
+          <b-form-group
+            class='mt-4'
+            label-for='banner-title-input'
+            :state='fieldState("banner.title")'
+            :label="$t('settings.application.banner.bannerTitle')"
+            v-if='settings.banner.enabled'
+          >
+            <b-form-input
+              id='banner-title-input'
+              v-model='settings.banner.title'
+              type='text'
+              :disabled='isBusy || viewOnly || !loaded'
+              :state='fieldState("banner.title")'
+            ></b-form-input>
+
+            <template slot='invalid-feedback'>
+              <div v-html="fieldError('banner.title')"></div>
+            </template>
+          </b-form-group>
+
+          <b-form-group
+            class='mt-4'
+            label-for='banner-message-input'
+            :state='fieldState("banner.message")'
+            :label="$t('settings.application.banner.message')"
+            v-if='settings.banner.enabled'
+          >
+            <b-form-textarea
+              id='banner-message-input'
+              v-model='settings.banner.message'
+              rows='3'
+              :disabled='isBusy || viewOnly || !loaded'
+              :state='fieldState("banner.message")'
+            ></b-form-textarea>
+
+            <template slot='invalid-feedback'>
+              <div v-html="fieldError('banner.message')"></div>
+            </template>
+          </b-form-group>
+
+          <b-form-group
```

Comment:
![image](https://user-images.githubusercontent.com/4281791/103874997-23dd6c00-50d2-11eb-8af5-410ab7795dbb.png)

Could be improve this feature to have the link as a button with the bootstrap varients, a text, link, target?